language_detector/main.py

```diff
@@ -1,9 +1,28 @@
-"""This is the entry point of the program."""
-
 from .languages import LANGUAGES
 
-
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    text_lang = "Unknown"
+    a_list = text.split()
+    
+    # For each language, check every word and see if it can be found in the
+    # languages.py file, if so add that word to a list.
+    # Finally, get the length of each of these lists.
+    try:
+        eng = [x for x in a_list for y in languages[2]['common_words'] if x==y]
+        length_english = len(eng)
+    except IndexError:
```

Comment:
Why is try/except used here? I don't quite see the logic where it would fail compared to German and Spanish.